grc/tests/test_qtbot.py

```diff
@@ -911,7 +911,7 @@ def drag():
     click_on(qtbot, qapp_cls_, n_sink.sinks[0])
     assert not fg_path.exists(), "File/Save (setup): .grc file already exists"
     assert not py_path.exists(), "File/Save (setup): Python file already exists"
-    menu_shortcut(qtbot, qapp_cls_, "build", QtCore.Qt.Key_B, QtCore.Qt.Key_G)
+    menu_shortcut(qtbot, qapp_cls_, "build", QtCore.Qt.Key_B, QtCore.Qt.Key_Enter)
```

Comment:
Seems to be the only functional change in this PR, and it seems to be logical you'd want to have "enter" on the menu itself to cause it to open.